src/templates/BlogPost/index.js

```diff
@@ -141,15 +117,24 @@ function BlogPost({ data, pageContext }) {
 }
 
 BlogPost.propTypes = {
-  data: PropTypes.shape({
-    mdx: PropTypes.shape(),
-  }).isRequired,
   pageContext: PropTypes.shape({
-    author: PropTypes.shape({
-      name: PropTypes.string.isRequired,
-      twitter: PropTypes.string,
-      twitterid: PropTypes.string,
-    }).isRequired,
+    blogPost: PropTypes.shape({
+      frontmatter: PropTypes.shape({
+        path: PropTypes.string.isRequired,
+        title: PropTypes.string.isRequired,
+        description: PropTypes.string.isRequired,
+        date: PropTypes.string.isRequired,
+        featuredImage: PropTypes.object.isRequired,
+        author: PropTypes.shape({
+          childAuthorsJson: PropTypes.shape({
```

Comment:
what's the need for the childAuthorsJson?

Reply:
It's from the `gatsby json` plugin, that allows us to query JSON files using the Gatsby schema. 'childAuthorsJson` is just the convention to work with said json files. We use those so we can link the posts with Markdown to the Authors. Check out a post `frontmatter` to check it out.